packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeModal/StakeModal.tsx

```diff
@@ -29,7 +29,12 @@ export const StakeModal = ({ onCancel }: StakeModalModalProps) => {
         <StakeEthFormContext.Provider value={stakeEthContextValues}>
             <NewModal
                 size="huge"
-                heading={<Translation id="TR_STAKE_ETH" />}
+                heading={
+                    <Translation
+                        id="TR_STAKE_NETWORK"
+                        values={{ networkSymbol: account.symbol.toUpperCase() }}
```

Comment:
```suggestion
                        values={{ symbol: account.symbol.toUpperCase() }}
```
please, otherwise @adderpositive will miss it

Reply:
This one is also ok. It is for a translation.

Reply:
I see. If the translation is new, please consider using `symbol` as @tomasklim suggested.

Reply:
Ok, I will unify the translations to use `symbol`. Most of the messages are new/changed